src/main/resources/templates/header.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org"
+      xmlns:sec="http://www.thymeleaf.org/thymeleaf-extras-springsecurity3">
+
+<body>
+
+<nav class="navbar navbar-inverse navbar-fixed-top" th:fragment="header">
+    <div class="container">
+        <div class="navbar-header">
+            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+                <span class="sr-only">Toggle navigation</span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+            </button>
+            <a class="navbar-brand" href="#">OCDU</a>
+        </div>
+        <div id="navbar" class="navbar-collapse collapse">
+            <ul class="nav navbar-nav">
+                <li class="active"><a href="/">Data</a></li>
+                <li><a href="patients">Patients</a></li>
+                <li><a href="events">Events</a></li>
+            </ul>
+            <ul class="nav navbar-nav pull-right">
+                <li>
+                    <form th:action="@{/logout}" method="post" style="padding-top: 8px;">
```

Comment:
You specified /logout RequestMapping as "get" on the backend 
 
 `     @RequestMapping(value="/logout", method= RequestMethod.GET)
         public String logout() {
             return "logout";
       }`
- why is method POST here?